spec/features/admin/questionnaire_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+feature 'questionnaire', js: true do
+  let(:division) { create(:division) }
+  let(:user) { create_member(division) }
+  let(:loan) { create(:loan, division: division) }
+  # let(:loan_question_set) { create(:loan_question_set, :loan_criteria) }
+
+  before do
+    login_as(user, scope: :user)
+    create(:loan_question_set, :loan_criteria)
+  end
+
+  context 'with conflicting changes' do
+    let(:response_set) { loan.create_criteria }
+
+    before do
+      response_set.summary = { text: 'dragon' }
```

Comment:
Can we do spaces for blocks and no spaces for hashes? Seems like a nice way to visually distinguish the two.